com.unity.render-pipelines.core/Editor/Lighting/LightAnchorEditor.cs

```diff
@@ -0,0 +1,662 @@
+                        EditorGUIUtility.SetWantsMouseJumping(0);
+                    }
+                    break;
+                case EventType.MouseDrag:
+                    if (GUIUtility.hotControl == id)
+                    {
+                        s_CurrentMousePosition = Event.current.mousePosition;
+                        Vector2 center = position;
+                        position = Handles.inverseMatrix.MultiplyPoint(s_CurrentMousePosition - s_DragScreenOffset);
+                        if (!Mathf.Approximately((center - position).magnitude, 0f))
+                        {
+                            GUI.changed = true;
+                        }
+                        Event.current.Use();
+                    }
+                    break;
+                case EventType.KeyDown:
+                    if (GUIUtility.hotControl == id && Event.current.keyCode == KeyCode.Escape)
+                    {
+                        position = Handles.inverseMatrix.MultiplyPoint(s_DragStartScreenPosition - s_DragScreenOffset);
+                        GUIUtility.hotControl = 0;
+                        GUI.changed = true;
+                        Event.current.Use();
+                    }
+                    break;
+            }
+
+            if (drawCapFunction != null)
+                drawCapFunction(id, position, Quaternion.identity, size, type);
+
+            return position;
+        }
+    }
+    class AngleFieldState
+    {
+        public float radius;
+        public Vector2 position;
+    }
+
+    static class LightAnchorStyles
+    {
+        static public float inspectorWidthPadding = 60f;
+        static public float presetButtonCount = 9f;
+        static public GUIStyle centeredLabel = GUI.skin.GetStyle("Label");
+        static public GUIContent presetTextureRimLeft = new GUIContent(Resources.Load<Texture2D>("PresetRim_Left"), "Rim Left");
+        static public GUIContent presetTextureKickLeft = new GUIContent(Resources.Load<Texture2D>("PresetKick_Left"), "Kick Left");
+        static public GUIContent presetTextureBounceLeft = new GUIContent(Resources.Load<Texture2D>("PresetBounce_Left"), "Bounce Left");
+        static public GUIContent presetTextureFillLeft = new GUIContent(Resources.Load<Texture2D>("PresetFill_Left"), "Fill Left");
+        static public GUIContent presetTextureHair = new GUIContent(Resources.Load<Texture2D>("PresetHair"), "Hair");
+        static public GUIContent presetTextureRimRight = new GUIContent(Resources.Load<Texture2D>("PresetRim_Right"), "Rim Right");
+        static public GUIContent presetTextureKickRight = new GUIContent(Resources.Load<Texture2D>("PresetKick_Right"), "Kick Right");
+        static public GUIContent presetTextureBounceRight = new GUIContent(Resources.Load<Texture2D>("PresetBounce_Right"), "Bounce Right");
+        static public GUIContent presetTextureFillRight = new GUIContent(Resources.Load<Texture2D>("PresetFill_Right"), "Fill Right");
```

Comment:
We cannot use Resources.Load in packages in Unity.
Look if CoreEditorUtils.LoadIcon can help for your case.

Reply:
Also, the issue is that what is in Resource folder is taken into the build. And we don't add uneeded weight in player builds.
So also move each resources outside or Resources folder please.